iocage/cli/fetch.py

```diff
@@ -45,51 +48,94 @@ def validate_count(ctx, param, value):
     else:
         return int(value)
 
+def release_prompt():
+    i = 0
+    default = None
+    for available_release in host.distribution.releases:
+        if available_release.name == host.release_version:
+            default = i
+            print(f"[{i}] \033[1m{available_release.name}\033[0m")
```

Comment:
In this case it's required to print to STDOUT because the script requires user interaction. If the user aborts the lines are not interesting in the log. Just one more misspelled shell command. In any other case you're right: we should use a logger that might print and/or log to files, remote logging services, etc.